neurom/check/tests/test_morphtree.py

```diff
@@ -110,28 +111,31 @@ def _genetate_tree_non_monotonic_section_boundary():
 
 
 def _generate_back_track_tree(n, dev):
-
```

Comment:
NeuroM v2 won't have a Section ctor so we need to get rid of its use in the tests.